.happy/terraform/modules/deletion/main.tf

```diff
@@ -0,0 +1,52 @@
+data aws_region current {}
```

Comment:
what's the deletion module for?

Reply:
It's an ECS task definition that deletes remote-dev db's when it's time to clean them up.